contrib/babelfishpg_tsql/sql/ownership.sql

```diff
@@ -617,4 +617,66 @@ GRANT EXECUTE ON PROCEDURE sys.babelfish_remove_domain_mapping_entry TO PUBLIC;
 
 CREATE OR REPLACE PROCEDURE sys.babelfish_truncate_domain_mapping_table()
   AS 'babelfishpg_tsql', 'babelfish_truncate_domain_mapping_table_internal' LANGUAGE C;
-GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
\ No newline at end of file
+GRANT EXECUTE ON PROCEDURE sys.babelfish_truncate_domain_mapping_table TO PUBLIC;
+
+CREATE TABLE sys.babelfish_extended_properties (
+  dbid smallint NOT NULL,
+  schema_name name NOT NULL,
+  major_name name NOT NULL,
+  minor_name name NOT NULL,
+  type sys.varchar(50) NOT NULL,
+  name sys.sysname NOT NULL,
+  orig_name sys.sysname NOT NULL,
+  value sys.sql_variant,
+  PRIMARY KEY (dbid, type, schema_name, major_name, minor_name, name)
+);
+GRANT SELECT on sys.babelfish_extended_properties TO PUBLIC;
+
+-- This view contains many outer joins that could potentially impair performance.
+-- To optimize this, it may be beneficial to begin by acquiring the object identifier (OID) and verifying permissions.
```

Comment:
It is not a-typical to see customer applications with thousands of extended properties being defined (see one with 57k of these). We should know whether this view is still practically usable whith 100, 1000, 10000 attributes.

Reply:
It’s surely usable with 100, 1000, 10000 attributes. In my test, it costs about 6 seconds to return 100000 attributes and 1 second to return 10000 attributes.

The comments here only follow Di's advise, to mark potentially performance decreasing.